### PR TITLE
Update tutorial for the new version

### PR DESCRIPTION
**Pull request by DavdGao**
## Description

1. Re-organize the section order in the tutorial. 
2. Add AgentScope Studio section in tutorial.
3. Add `project` parameters in each example to fit AgentScope Studio.

> Note: This PR should be merged after the AgentScope Studio PR, but is ready for review now!

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

**Comment by DavdGao** [comment]
> LGTM, but a minor question: which PR contains the update of README file in web dir?

Update in the studio section.

File: README.md [hunk 1]
```diff
@@ -45,15 +47,15 @@ https://github.com/DavdGao/AgentScope/assets/102287034/d9acf581-20d2-4294-8d10-9
...
 
 - **[2024-04-06]** We release **AgentScope** v0.0.3 now!
 
-- **[2024-04-06]** New examples [Gomoku](https://github.com/modelscope/agentscope/blob/main/examples/game_gomoku), [Conversation with ReAct Agent](https://github.com/modelscope/agentscope/blob/main/examples/conversation_with_react_agent), [Conversation with RAG Agent](https://github.com/modelscope/agentscope/blob/main/examples/conversation_with_RAG_agents) and [Distributed Parallel Search](https://github.com/modelscope/agentscope/blob/main/examples/distributed_search) are available now!
+- **[2024-04-06]** New examples [Gomoku](https://github.com/modelscope/agentscope/blob/main/examples/game_gomoku), [Conversation with ReAct Agent](https://github.com/modelscope/agentscope/blob/main/examples/conversation_with_react_agent), [Conversation with RAG Agent](https://github.com/modelscope/agentscope/blob/main/examples/conversation_with_RAG_agents) and [Distributed Parallel Search](https://github.com/modelscope/agentscope/blob/main/examples/distributed_parallel_optimization) are available now!
```
Comment (pan-x-c) [inline README.md:58]: Search -> Optimization

File: README_ZH.md [hunk 2]
```diff
@@ -136,7 +138,7 @@ AgentScope支持使用以下库快速部署本地模型服务。
 - 分布式
   - [分布式对话](./examples/distributed_conversation)
   - [分布式辩论](./examples/distributed_debate)
-  - [分布式并行搜索](./examples/distributed_search)
+  - [分布式并行搜索](./examples/distributed_parallel_optimization)
```
Comment (pan-x-c) [inline README_ZH.md:141]: 搜索 -> 优化

File: README_ZH.md [hunk 3]
```diff
@@ -36,15 +38,15 @@
...
 
-- <img src="https://img.alicdn.com/imgextra/i3/O1CN01SFL0Gu26nrQBFKXFR_!!6000000007707-2-tps-500-500.png" alt="new" width="30" height="30"/>**[2024-04-27]** [AgentScope Workstation](https://agentscope.aliyun.com/)上线了！ 欢迎使用 Workstation 体验如何在*拖拉拽编程平台* 零代码搭建多智体应用，也欢迎大家通过*copilot*查询AgentScope各种小知识！
+- **[2024-04-27]** [AgentScope Workstation](https://agentscope.aliyun.com/)上线了！ 欢迎使用 Workstation 体验如何在*拖拉拽编程平台* 零代码搭建多智体应用，也欢迎大家通过*copilot*查询AgentScope各种小知识！
 
 - **[2024-04-19]** AgentScope现已经支持Llama3！我们提供了面向CPU推理和GPU推理的[脚本](./examples/model_llama3)和[模型配置](./examples/model_llama3)，一键式开启Llama3的探索，在我们的样例中尝试Llama3吧！
 
 - **[2024-04-06]** 我们现在发布了**AgentScope** v0.0.3版本！
 
-- **[2024-04-06]** 新的样例“[五子棋](./examples/game_gomoku)”，“[与ReAct智能体对话](./examples/conversation_with_react_agent)”，“[与RAG智能体对话](./examples/conversation_with_RAG_agents)”，“[分布式并行搜索](./examples/distributed_search)”上线了！
+- **[2024-04-06]** 新的样例“[五子棋](./examples/game_gomoku)”，“[与ReAct智能体对话](./examples/conversation_with_react_agent)”，“[与RAG智能体对话](./examples/conversation_with_RAG_agents)”，“[分布式并行搜索](./examples/distributed_parallel_optimization)”上线了！
```
Comment (pan-x-c) [inline README_ZH.md:49]: 搜索 -> 优化

File: docs/sphinx_doc/en/source/tutorial/main.md [hunk 4]
```diff
@@ -10,26 +10,23 @@ AgentScope is an innovative multi-agent platform designed to empower developers
...
-
-- [Customizing Your Own Agent](201-agent-en)
-- [Pipeline and MsgHub](202-pipeline-en)
-- [Model](203-model-en)
-- [Service](204-service-en)
-- [Memory](205-memory-en)
-- [Prompt Engineering](206-prompt-en)
-- [Monitor](207-monitor-en)
-- [Distribution](208-distribute-en)
+- [About AgentScope](101-agentscope.md)
```
Comment (pan-x-c) [inline docs/sphinx_doc/en/source/tutorial/main.md:13]: `xxx-en` should be used to distinguish English and Chinese version

**Review by pan-x-c — COMMENTED** [review]
Please see the inline comments, others LGTM

**Review by pan-x-c — APPROVED** [review]
LGTM

**Pull request merged**